types/localized-countries/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for localized-countries 1.0
+// Project: https://github.com/marcbachmann/localized-countries#readme
+// Definitions by: coderslagoon <https://github.com/coderslagoon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface LocalizedCountries {
+    get(code: string): string;
+    array(): Array<{
+        code: string;
+        label: string;
+    }>;
+    object(): { [code: string]: string };
+}
+
+declare function localizedCountries(locale: string|{}): LocalizedCountries;
```

Comment:
did you mean `object` instead of `{}`? `{}` accepts all types including `number` for instance. `object` requires `/// TypeScript version: 2.2`